package.json

```diff
@@ -19,11 +19,13 @@
     "build"
   ],
   "scripts": {
-    "start": "tsc && node ./src/index.js",
-    "dev": "tsc && cozy-run-dev",
-    "standalone": "tsc && cozy-run-standalone",
+    "start": "node ./src/index.js",
```

Comment:
Is this because you don't like typescript or just a mistake? 😄

Reply:
A mistake of course! Thanks for the review